src/trackers/COMMON.py

```
@@ -68,29 +68,36 @@ async def edit_torrent(self, meta, tracker, source_flag, torrent_filename="BASE"
-                        return None
+                async with httpx.AsyncClient(headers=headers, params=params, timeout=30.0) as session:
+                    async with session.stream("GET", downurl) as r:
+                        r.raise_for_status()
+                        async with aiofiles.open(path, "wb") as f:
+                            async for chunk in r.aiter_bytes():
+                                await f.write(chunk)
+
+                if hash_is_id:
+                    torrent_hash = await self.get_torrent_hash(meta, tracker)
+                    return torrent_hash
+                return None
```

Ruff (0.14.6)

Consider moving this statement to an `else` block

(TRY300)